test/dynamo/test_trace_rules.py

```diff
@@ -395,21 +332,6 @@ def test_skipfiles_inlinelist(self):
     def test_torch_name_rule_map_updated(self):
         # Generate the allowed objects based on heuristic defined in `allowed_functions.py`,
         objs = gen_allowed_objs_and_ids(record=True, c_binding_only=True)
-        # Test ctx manager classes are updated in torch_name_rule_map.
-        generated = objs.ctx_mamager_classes
-        used = set()
-        for x in (
-            set(torch_ctx_manager_classes.keys()) | ignored_ctx_manager_class_names
-        ):
-            obj = load_object(x)
-            if obj is not None:
-                used.add(obj)
-        self._check_set_equality(
-            generated,
-            used,
-            "torch_ctx_manager_classes",
-            "ignored_ctx_manager_class_names",
-        )
```

Comment:
Removed this test which is used to ensure ctx manager classes are updated, since they can't be supported automatically even we find there is a new ctx manager.